executor/common.h

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016 syzkaller project authors. All rights reserved.
+// Copyright 2020 syzkaller project authors. All rights reserved.
```

Comment:
There is still something wrong here. Try to somehow restore the old contents.

Reply:
Do the leave the year as 2016 ?

Reply:
Yes. We don't update them.
Or otherwise it should be a global change once per year. But lots of projects don't do this and I don't any benefit in increased diffs.